compiler/testData/codegen/bytecodeListing/jvmStatic/jvmStaticExternalWithDeclarationRewrite.kt

```diff
@@ -0,0 +1,15 @@
+// WITH_STDLIB
+// REWRITE_JVM_STATIC_IN_COMPANION
+// JVM_TARGET: 1.8
+
+object TestObject {
```

Comment:
Add a test with `@JvmStatic` function accessing private properties of the object.